cf-nodejs-logging-support-core/log-core.js

```diff
@@ -96,6 +96,16 @@ var init = function () {
     }
 };
 
+var JSONparseSave = function (value) {
```

Comment:
you could extract the null check into this function as well.

Reply:
Yes that would clean up the function, good catch